python/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+
+IF(NOT Kokkos_ENABLE_PYBIND11)
+    return()
+ENDIF()
+
+
+# don't build a static python module
+SET(BUILD_SHARED_LIBS ON)
+
+# pybind11 has not migrated to CMAKE_CXX_STANDARD and neither has kokkos
+if(NOT Kokkos_CXX_STANDARD)
+    set(Kokkos_CXX_STANDARD 11)
```

Comment:
This should already default to 11. Was there a reason for adding this?

Reply:
Yea it was showing up as empty